src/util/find_uid.c

```diff
@@ -165,6 +166,23 @@ static errno_t get_uid_from_pid(const pid_t pid, uid_t *uid)
         return EINVAL;
     }
 
+    /* Get process name. */
+    if (strncmp(buf, "Name:\t", 6) != 0) {
```

Comment:
Hi,

do you know if it is guaranteed that the status file will always start with `Name:`? Since kernel is very strict about changes visible in user-space I doubt this will ever change but who knows.

bye,
Sumit

Reply:
I changed it to strstr to be on a safe side, though I would say it is safe to assume it will be always on the top.